HACKING.md

```diff
@@ -22,6 +22,34 @@ run everything under a Visual Studio Tools Command Prompt (or after
 running `vcvarsall` in a normal command prompt).  See below if you
 want to use mingw or some other compiler instead of Visual Studio.
 
+Assuming that you have python installed, then the steps for building under
+ Windows using Visual Studio are:
+ 
+In a bash window, clone and checkout the latest release
```

Comment:
Sorry, I meant up here. Being explicit seems fine, but this will work just as well in cmd as in bash. So maybe just "Clone and checkout the..."